auth-web/src/util/constants.ts

```diff
@@ -435,7 +435,8 @@ export enum LDFlags {
     SentryEnable = 'sentry-enable',
     SupportRestorationEntities = 'supported-restoration-entities',
     AllowableBusinessSearchTypes = 'allowable-business-search-types',
-    AllowableBusinessPasscodeTypes = 'allowable-business-passcode-types'
+    AllowableBusinessPasscodeTypes = 'allowable-business-passcode-types',
+    EnableAffiliationDelegation = 'enable-affiliation-delegation'
```

Comment:
Not sorting these alphabetically, I'd rather leave them so the more recent ones are at the bottom. That way when we do cleanup - we know which flags are the oldest.